packages/lodestar/src/opPool/opPool.ts

```diff
@@ -82,35 +78,7 @@ export class OpPool extends EventEmitter {
       this.depositData.removeOld(signedBlock.message.body.eth1Data.depositCount),
       this.proposerSlashings.remove(signedBlock.message.body.proposerSlashings),
       this.attesterSlashings.remove(signedBlock.message.body.attesterSlashings),
-      this.aggregateAndProofs.removeIncluded(signedBlock.message.body.attestations),
-      this.checkDuplicateProposer(signedBlock)
+      this.aggregateAndProofs.removeIncluded(signedBlock.message.body.attestations)
     ]);
   }
-
```

Comment:
why do we remove the duplicate check here, any replacement for that?

Reply:
It's invalid check that made validator not propose twice in same epoch(which is case when you have small amount of validators), proposerSlashing is only if validator signed two blocks for sam e slot. Also this check should be added to validate method of block processing so it checks blocks from network